app/components/record-delete-alert-dialog.tsx

```diff
@@ -31,9 +34,13 @@ export default function RecordDeleteAlertDialog(props: RecordDeleteAlertDialogPr
             <Button colorScheme="gray" onClick={onCancel} ref={cancelRef}>
               Cancel
             </Button>
-            <Button colorScheme="brand" onClick={onConfirm} ml="3">
-              Delete
-            </Button>
+            <Form method="delete" style={{ margin: 0 }}>
+              <input type="hidden" name="id" value={dnsRecord?.id} />
+              <input type="hidden" name="intent" value="delete-record" />
```

Comment:
I pretty sure Chakra UI Input provides some nice styling, unlike the input which has basic styles. Since inputs above have type `hidden` there is no need for them to be fancy cuz they are not visible